authors/apps/social_auth/serializers.py

```diff
@@ -0,0 +1,73 @@
+from rest_framework import serializers
+
+from authors.apps.social_auth import google, facebook
+from authors.apps.social_auth.common import create_user_and_return_token
+
+
+# noinspection PyMethodMayBeStatic
+class GoogleSocialAuthViewSerializer(serializers.Serializer):
+    """ Handles all social auth related tasks from google """
+
+    def create(self, validated_data):
```

Comment:
Why do we need this @reiosantos ?

Reply:
**`<Serializer>`** is an abstract class, and those two methods are defined in that class. So if I inherit from an abstract class, then I must implement all the methods in it. Otherwise, I must declare the class abstract. **`<pylint>`** was complaining about that implementation, and I wanted to enforce those coding conventions. So that's why I have them defined but not used.